### ISSUE TITLE
Issue when scrolling tables with frozen columns

### ISSUE THREAD

**Issue by Martino-Martinez** [olifolkerd/tabulator#3716]
Working example:

[Fiddle](https://jsfiddle.net/kc1f9eha/)

Issue:

- scroll right
- scroll down a few pages
- scroll left
- scroll up

You will see that some columns are not in the correct place. If you scroll left/right just a little bit, the display is updated and correct again.

![screenshot](https://user-images.githubusercontent.com/103148068/165083611-85bafc47-9823-42ff-87a9-95d9ac85a12b.PNG)

-- Martin

**Comment by olifolkerd** [comment]
@Martino-Martinez please use the bug issue template when creating a bug report, it will prompt you to provide all the information we need to help you out

**Comment by Martino-Martinez** [comment]
**Hi Oli -**

here you go...

**Describe the bug**

- Display issue when scrolling tables with frozen columns

**Tabulator Info**
- V 5.2.2

**Working Example**

- https://jsfiddle.net/kc1f9eha/

**To Reproduce**

- Scroll right
- Scroll down a few pages
- Scroll left
- Scroll up
- You will see that some columns are not in the correct place. If you scroll left/right just a little bit, the display is updated and correct again.

**Expected behavior**

- All columns should be in the correct place :-)

**Screenshots**

![screenshot](https://user-images.githubusercontent.com/103148068/165156850-93b13681-2a77-4b90-9c76-ad9e76de40fa.PNG)

**Desktop (please complete the following information):**

 - OS: Windows 10
 - Browser: Chrome
 - Version: 100.0.4896.127

-- Martin

**Comment by blankbadge** [comment]
Hi Oli, Martin,

I'm glad to see someone also encounterd this issue, as I was having a lot of difficulty reproducing on a small example.  But looking at Martin's example I was able to tweak mine to also show the issue.

https://jsfiddle.net/blankbadge/wyos7cvr/

Initially the table looks like this with two frozen columns and then eight normal columns:

![image](https://user-images.githubusercontent.com/57735852/166076497-ddc9200c-e0e0-49d4-8cad-fbe3394fe6b7.png)

To reproduce:

(1) Use your mouse to click and drag the vertical scrollbar all the way to the bottom of the table
(2) Use your mouse to click and drag the horizontal scrollbar all the way to the right of the table
(3) Use your mouse to click and drag the horizontal scrollbar about halfway back towards the left.  It does not happen if you bring the horizontal scrollbar all the way back to the left!
(4) Now use your mouse to click and drag the vertical scrollbar back up towards the top.  Eventually you will see the frozen columns become misaligned like this:

![image](https://user-images.githubusercontent.com/57735852/166076799-3b7301ca-586b-4015-a20b-1741d319a230.png)

I noticed that the misformatting of frozen columns seems to begin once you get past the rows that are still in the vertical virtual dom buffer after scrolling to the bottom.  If I set renderVerticalBuffer to a small value like 1, I encounter the rows with misaligned frozen columns more quickly when scrolling back up than with the default buffer size.

Thanks!
-blankbadge
